### ISSUE TITLE
fix: facing multiple issues with 2.7.0 multiple stripe account feature

### ISSUE THREAD

**Issue by ravinderk** [impress-org/givewp#4781]
# Bug Report

## User Story

As an admin, I want multiple stripe account feature to work fine so that setup it easily and accept the donation.

## Details

I am facing multiple issues:
- [x] Update and Edit account name for the manual key does not work fine.
![image](https://user-images.githubusercontent.com/1784821/83291020-471f6f00-a205-11ea-9779-b097c2e95b86.png)

- [x] I get the option to add a manual stripe key even if this feature disable.
![image](https://user-images.githubusercontent.com/1784821/83291387-e3e20c80-a205-11ea-926f-779c629214f1.png)

- [x] Do not show an alert when updating the manual stripe key.
![image](https://user-images.githubusercontent.com/1784821/83291569-2e638900-a206-11ea-8f36-d93938cc6ad6.png)

- [x]  Do not show an alert when disconnecting the manual stripeaccount.
![image](https://user-images.githubusercontent.com/1784821/83291640-489d6700-a206-11ea-94e0-748a6672cf0d.png)

- [x] Always keep the first manual key account as default when adding only manual keys
![image](https://user-images.githubusercontent.com/1784821/83291913-d24d3480-a206-11ea-82e7-d4393adf9163.png)

- [x] Styling issue.
![image](https://user-images.githubusercontent.com/1784821/83292034-0cb6d180-a207-11ea-8888-c23551727e57.png)

- [x] Can you confirm that name will be attached to a manual key account because I can not see names.
![image](https://user-images.githubusercontent.com/1784821/83292404-a54d5180-a207-11ea-9610-d1f52a5713cb.png)
![image](https://user-images.githubusercontent.com/1784821/83292431-b302d700-a207-11ea-908b-3acd27cbec06.png)

- [x] Show globally selected default account as default account in per-form settings.
![image](https://user-images.githubusercontent.com/1784821/83292600-02490780-a208-11ea-9e90-30af92a55cca.png)
![image](https://user-images.githubusercontent.com/1784821/83292612-08d77f00-a208-11ea-97bc-07dad6766db2.png)

- [x] Store Stripe account information like name in donation data for future reference: https://github.com/impress-org/givewp/pull/4662#issuecomment-623706801

## Expected Behavior

Multiple Stripe account feature must be bug-free :D

## Bug Type

These bugs only exist in the upcoming Give 2.7.0.

## Acceptance Criteria

- [x] The Issues mentioned in `Details` section have been fixed and functionality is working as expected.

**Comment by mehul0810** [comment]
@ravinderk

> Can you confirm that name will be attached to a manual key account because I can not see names.
- As we discussed, for manual API keys, we are identifying the name of the Stripe account via the account slug. so, there will not be any entry for the account name under the Stripe account array. However, the issue related to this is generated due to EDIT, UPDATE, and CANCEL account name. So, I have fixed it. But, I will let you know tomorrow once all the pending points are completed.

> Do not show an alert when updating the manual stripe key.
> Do not show an alert when disconnecting the manual stripeaccount.
- We don't have much control to restrict a specific admin setting fields change event to don't show save before leaving the page popup. Check the PR for more information: https://github.com/impress-org/givewp/pull/2067


I will fix the remaining points tomorrow and let you know.
